addons/l10n_sg_ubl_pint/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
do we still need those on new files, i have the feeling that we don't but i may be mistaken :thinking:

Reply:
Probably yes, the one we don't need is the utf-8 thing.